warehouse/query-core/src/test/java/datawave/query/jexl/functions/ContentFunctionsTest.java

```diff
@@ -337,13 +358,14 @@ public void testEvaluation2() {
         list1 = asList(1, 2, 3);
         list2 = asList(5, 6, 7);
         
-        termOffSetMap.put("dog", new TermFrequencyList(Maps.immutableEntry(new Zone("CONTENT", true, eventId), list1)));
-        termOffSetMap.put("cat", new TermFrequencyList(Maps.immutableEntry(new Zone("CONTENT", true, eventId), list2)));
+        termOffSetMap.putTermFrequencyList("dog", new TermFrequencyList(Maps.immutableEntry(new Zone("CONTENT", true, eventId), list1)));
+        termOffSetMap.putTermFrequencyList("cat", new TermFrequencyList(Maps.immutableEntry(new Zone("CONTENT", true, eventId), list2)));
         
         context.set(Constants.TERM_OFFSET_MAP_JEXL_VARIABLE_NAME, termOffSetMap);
         Object o = expr.evaluate(context);
         
         Assert.assertTrue(expect(o, false));
+        assertPhraseOffsetsEmpty();
```

Comment:
good.  OK, I am going to stop checking each one now.  I think I am convinced that you are pulling the offsets from the correct places.

Reply:
If there is one that you are particularly concerned about, point it out for me please.